crates/volta-migrate/src/v1.rs

```diff
@@ -87,17 +87,18 @@ impl TryFrom<V0> for V1 {
 
             debug!("Removing old Volta binaries");
             let old_volta_bin = new_home.root().join("volta");
-            if old_volta_bin.exists() {
-                remove_file(&old_volta_bin).with_context(|| ErrorKind::DeleteFileError {
+
+            remove_file(&old_volta_bin)
+                .or_else(ok_if_not_found)
```

Comment:
Instead of directly doing `remove_file().or_else()`, there actually is a helper in `volta_core::fs` named `remove_file_if_exists`, so we should re-use that (it internally does the same thing since it's a common pattern).